CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ### 🐞 Fixed
-- SPM dependency for SwiftProtobuf was outdated and sometimes Xcode couldn't fetch the latest required version. [#730](https://github.com/GetStream/stream-video-swift/pull/730)
+- During a reconnection/migration the current user will not be appearing twice any more. [#731](https://github.com/GetStream/stream-video-swift/pull/731)
```

Comment:
Is this correct? 🤔

Reply:
as this will be merged after the release, that will become a merge conflict and the PR won't be able to be merged without it being resolved.